gradle/forge.gradle

```diff
@@ -162,21 +159,24 @@ dependencies {
     runtimeOnly fg.deobf("slimeknights.mantle:Mantle:1.16.5-"+project.mantle_version)
 
     // Survive - https://www.curseforge.com/minecraft/mc-mods/survive
-    implementation fg.deobf("curse.maven:survive-367337:"+project.survive_fileid)
+    compileOnly fg.deobf("curse.maven:survive-367337:"+project.survive_fileid)
```

Comment:
running with survive throws a mixin inject error for me. With old and new version